repo/packages/D/datastax-dse/600/package.json

```
@@ -0,0 +1,29 @@
+{
+  "packagingVersion": "4.0",
+  "name": "datastax-dse",
+  "version": "3.1.0-6.7.6",
+  "upgradesFrom": [
+    "2.4.0-5.1.10",
+    "3.0.0-6.7.2"
+  ],
+  "downgradesTo": [
+    "2.4.0-5.1.10",
+    "3.0.0-6.7.2"
+  ],
+  "maintainer": "techpartner@datastax.com",
+  "minDcosReleaseVersion": "1.11",
+  "description": "Datastax Enterprise",
+  "selected": true,
+  "framework": true,
+  "tags": [
+    "datastax",
+    "dse",
+    "database",
+    "enterprise",
+    "nosql"
+  ],
+  "preInstallNotes": "This DC/OS Service is maintained by DataStax and available to mutual DataStax and Mesosphere customers (eval and production) and prospects (eval) .\n\nFunctionality supported in Opscenter includes Repair, Monitoring, Backup & Restore, as well as some basic admin tools including cleanup, restart, and drain. No other OpsCenter functionality is tested or supported within the DC/OS Integration. Please make any feature requests to techpartner@datastax.com\n\nDefault configuration requires 3 agent nodes each with: 2.5 CPU | 8500 MB MEM | 11240 MB Disk\n\nContact Mesosphere and DataStax before deploying this service to ensure proper support channels are made available. Contact your rep at either company or techpartner@datastax.com. By using this service you agree to the license terms here: https://gist.github.com/012345/d1df9d43a0dfaa49e628c7bfbc9ffb52\n\nVersions: DataStax Enterprise 6.7.6 (Standard and Max) & OpsCenter 6.7.6",
+  "postInstallNotes": "DC/OS Datastax Enterprise is being installed!\n\n\tDocumentation: https://docs.datastax.com\n\tIssues: https://www.datastax.com/contactus",
+  "postUninstallNotes": "DC/OS Datastax Enterprise is being uninstalled.\n\nFor DC/OS versions from 1.11 no further action is required. For older DC/OS versions follow the instructions at https://docs.d2iq.com/mesosphere/dcos/services/dse/stub-universe/uninstall/ to remove any persistent state if required.",
+  "lastUpdated": 1576524133847
```

```suggestion
  "lastUpdated": 1576524133
```
epoch time is stored in seconds. See previous value:
```
-  "lastUpdated": 1571120354
```